gke/README.md

```diff
@@ -51,8 +51,34 @@ The following describes the setup process for this project:
 1. Install Jenkins on GKE using the stable/jenkins helm chart: [Jenkins On GKE Tutorial](
    https://cloud.google.com/solutions/jenkins-on-kubernetes-engine-tutorial).
 
-1. Install and configure the GKE plugin: [Jenkins GKE Docs](
-   https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md).
+1. Install and configure the following plugins:
+   1. GKE plugin: [Jenkins GKE Docs](https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md).
+   1. GCS plugin: [Jenkins GCS Docs](https://github.com/jenkinsci/google-storage-plugin/blob/develop/README.md).
+
+1. Create a GCS bucket to upload your test work, e.g. 'projectname-jenkins-test-bucket'
+
+1. Create GCP service account that can push objects to your bucket and containers to your registry:
+   ```bash
+    export PROJECT=YOURGOOGLEPROJECTNAME
+    export SA=kaniko-role
+    gcloud iam service-accounts create $SA
+    export SA_EMAIL=$SA@$PROJECT.iam.gserviceaccount.com
+    gcloud projects add-iam-policy-binding $PROJECT \
+    --member serviceAccount:$SA_EMAIL \
+    --role ‘roles/containeranalysis.admin’
+    gcloud projects add-iam-policy-binding $PROJECT \
+    --member serviceAccount:$SA_EMAIL \
+    --role ‘roles/storage.objectAdmin'
+    gcloud iam service-accounts keys create ./kaniko-secret.json --iam-account $SA_EMAIL
+    ```
+
+1. Add the new credential to your jenkins project:
```

Comment:
The kaniko secret doesn't need to be added to the Jenkins cred store, as it's only referenced via the kaniko-pod.yaml which pulls is directly from the k8s secret store.